staticjinja/staticjinja.py

```diff
@@ -93,39 +117,40 @@ def get_template(self, template_name):
         """
         return self._env.get_template(template_name)
 
-    def _get_context_generator(self, template_name):
-        """Get a context generator for a template.
-
-        Raises a :exc:`ValueError` if no matching context generator can be
-        found.
-
-        :param template_name: the name of the template
-        """
-        for regex, context_generator in self.contexts:
-            if re.match(regex, template_name):
-                return context_generator
-        raise ValueError("no matching context generator")
```

Comment:
I'm a little worried that removing this is backwards compatible because someone may be subclassing this class and overriding this. Doubtful though, looks like I did so to avoid some ugly code in `get_context()`. Just want to note for posterity.